tests/test_templatestags.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+from django.test import TestCase
+from django import template
+from request.templatetags.request_admin import trunc, pie_chart
+from request.templatetags.request_tag import ActiveUserNode, active_users
+
+
+class RequestAdminTruncTest(TestCase):
+    def test_not_truncate(self):
+        string = "A" * 10
+        new_string = trunc(string, 20)
+        self.assertEqual(len(new_string), 10)
+
+    def test_truncate(self):
+        string = "A" * 10
+        new_string = trunc(string, 5)
+        self.assertLess(len(new_string), 6)
+
+
+class RequestAdminPieChart(TestCase):
+    def test_pie_chart(self):
+        inventory = ['apple', 'lemon', 'apple', 'orange', 'lemon', 'lemon']
+        pie_chart(inventory)
+
+
+# TODO: It's unused but add a parser: template.debug.DebugParser
+class RequestTagActiveUserNodeTest(TestCase):
+    def test_syntax_error_bad_args_number(self):
+        token = template.base.Token(2, 'active_users foo')
+        self.assertRaises(template.TemplateSyntaxError, ActiveUserNode, None, token)
+
+    def test_in_amount_duration_as_varname(self):
+        token = template.base.Token(2, "active_users in 10 minutes as user_list")
+        ActiveUserNode(None, token)
```

Comment:
What about asserting that `user_list` was set as the `as_varname`, I'd suggest using something other than `user_list` here since it's the default value.